packages/demo/src/demo-building-blocs/ScrollTop.tsx

```diff
@@ -14,4 +14,4 @@ const ScrollToTop: React.FunctionComponent<RouteComponentProps> = ({history}) =>
     return null;
 };
 
-export default withRouter(ScrollToTop);
+export default withRouter(ScrollToTop) as any;
```

Comment:
Since Router v6 will change this files and change the export, i'll leave this as any for now.